test/store.test.js

```diff
@@ -254,3 +258,21 @@ test('Store#setFeatureProperty', t => {
   t.end();
 });
 
+test('Store#storeAndRestoreMapConfig', t => {
+  const map = createMap();
+  // Disable doubleClickZoom
+  map.doubleClickZoom.disable();
+  // Check it's disabled
+  t.equal(map.doubleClickZoom.isEnabled(), false, 'Disables doubleClickZoom on the map');
+  const ctx = { map: map };
+  const store = new Store(ctx);
+  store.storeMapConfig();
+  // Check we can get the initial state of it
+  console.log(store);
```

Comment:
Hi! I noticed this on my test output.

Reply:
Oh, crap. This is already merged. Should I create another PR to fix?

Reply:
That would be great. Thanks! Can you look at adding no console.log to the eslint config as well?

Reply:
This has been done, see: https://github.com/mapbox/mapbox-gl-draw/pull/716